documentation/SDKs/javascript/README.md

```diff
@@ -0,0 +1,358 @@
+#JavaScript SDK for Nether
+
+## Work in Progress
+
+## Downloading the JavaScript SDK for Nether
+
+### Installing the JavaScript SDK for Nether
+
+## Usage
+
+### Analytics
+
+#### Active sessions - daily
+You can use the ***Nether.Analytics.GetActiveSessionsDaily*** method to get the total active sessions per day. Returns an array of ActiveSession
```

Comment:
@theciscokid - I agree, and like the idea of rippling that change through to the APIs :-)

As @krist00fer points out, it probably doesn't make sense to include these APIs in the initial client SDK, so let's defer this discussion for now. The initial analytics client SDK should allow sending analytics events